statsmodels/sandbox/nonparametric/kernel_extras.py

```diff
@@ -131,7 +131,7 @@ def _compute_test_stat(self, u):
         n = np.shape(u)[0]
         XLOO = LeaveOneOut(self.exog)
         uLOO = LeaveOneOut(u[:,None]).__iter__()
-        I = 0
+        ival = 0
```

Comment:
`ival` is not more informative than `i`

Reply:
Yah, any other suggestions?  I don't have the textbook listed as a reference on hand.

`I_scalar` follows a pattern set by `I_dist` in the previous method.
`f_sum` since it is a running total of `fi.sum()` terms